calendar-bundle/src/Resources/contao/modules/ModuleEventlist.php

```diff
@@ -70,7 +70,7 @@ public function generate()
 		}
 
 		// Show the event reader if an item has been selected
-		if ($this->cal_readerModule > 0 && isset($_GET['auto_item']))
+		if ($this->cal_readerModule > 0 && Input::get('auto_item') !== null)
```

Comment:
This was probably used to not mark `auto_item` as _being used_, how will that work in the future?

Reply:
As it is only executed if `cal_readerModule` is set, this should not be a problem here.

To ensure backwards compatibility we could add the following line here:

```php
Input::setUnusedRouteParameters(['auto_item', ...Input::getUnusedRouteParameters()]);
```

WDYT?

Reply:
I don‘t think that we are using `isset($_GET[])` to work around unused route parameters here. The pattern is used quite often throughout the code base, e.g. `isset($_GET['table']) && \in_array(Input::get('table'), $arrTables)` in the back end (where there are no unused route parameters).